package.json

```diff
@@ -49,33 +49,43 @@
     ]
   },
   "dependencies": {
-    "axios": "~0.21.1",
+    "@babel/core": "~7.9.0",
```

Comment:
babel-core is now deprecated as it uses v6 as opposed to @babel/corev7. I added it somewhere while installing packages because of babel-loader probably. I actually have another branch that updates @babel/core further and changes the config file as such. For now, this does not actually do anything, but later on when I merge my other branch @babel/core will replace babel-core

Reply:
Okay. We can wait for the next update on another PR.